src/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import { Provider } from 'react-redux';
+import { store } from './app/store';
+import App from './app/App';
+import reportWebVitals from './reportWebVitals';
+import './index.css';
+
+const container = document.getElementById('root');
+const root = createRoot(container);
+
+root.render(
+  <React.StrictMode>
+    <Provider store={store}>
+      <App />
+    </Provider>
+  </React.StrictMode>,
+);
+
+// If you want to start measuring performance in your app, pass a function
+// to log results (for example: reportWebVitals(console.log))
+// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
+reportWebVitals();
```

Comment:
I think it should be better if we remove `reportwebvitals()` function as long as we don't need it.
If you decide to delete it, you should also delete `reportWebVitals.js` file

Reply:
@RWUBAKWANAYO noted, thank you.